src/audio/volume/volume.c

```diff
@@ -750,6 +766,8 @@ static int volume_copy(struct comp_dev *dev)
 
 	/* Get source, sink, number of frames etc. to process. */
 	comp_get_copy_limits_with_lock(source, sink, &c);
+	/* limit frames to be divided by 4 for alignment of 8-byte */
+	c.frames &= ~0x03;
```

Comment:
Can we get a ```comp_get_copy_limits_with_lock_aligned(source, sink, &c, alignment);``` API call as this will be used in other places too.

Reply:
@singalsu fyi - this could be a new PR though. @andrula-song let do this change in a new PR if you dont have time before CNY.

Reply:
That's a good idea, I can change other components to use it instead of their own adjust of frames and bytes

Reply:
I can add the comp_get_copy_limits_with_lock_aligned() API before CNY, but change other components maybe need more time.